src/main/java/edu/neu/EDE/ExcelDataExtractor.java

```diff
@@ -43,4 +46,16 @@ private void readFile(File f) throws IOException {
             invalidFiles.add(f.getAbsolutePath());
         }
     }
+
+    private void writeFiles() {
+        WorkbookWriter writer = new WorkbookWriter();
+        writer.setSheetType(DataType.STATISTIC);
+        writer.setColumnType(DataType.STIMULUS);
+        writer.setRowType(DataType.SUBJECT);
+        writer.setData(slideMetricData);
```

Comment:
yes, this is terribly ugly, I know.  This is essentially throwaway code, to be completely overhauled when we have a gui.  Not worth fixing right now